From 544ea41d58c1feff47832fdbff3398c626398d59 Mon Sep 17 00:00:00 2001
From: Andrew Steinborn <git@steinborn.me>
Date: Tue, 8 Oct 2019 01:23:37 -0400
Subject: [PATCH] Fix JLine issue on Windows

JLine 0.9.x apparently likes to use the implementation version when
creating temporary files. Change our implementation version to omit
double quotes.

diff --git a/pom.xml b/pom.xml
index 30915c6a..b5e1875e 100644
--- a/pom.xml
+++ b/pom.xml
@@ -104,6 +104,7 @@
   <build>
     <defaultGoal>clean install</defaultGoal>
     <plugins>
+      <!-- Canyon - Replace gitdescribe with scriptus
       <plugin>
         <groupId>com.lukegb.mojo</groupId>
         <artifactId>gitdescribe-maven-plugin</artifactId>
@@ -120,7 +121,24 @@
             </goals>
           </execution>
         </executions>
+      </plugin> -->
+      <plugin>
+        <groupId>net.md-5</groupId>
+        <artifactId>scriptus</artifactId>
+        <version>0.3.1</version>
+        <executions>
+          <execution>
+            <phase>initialize</phase>
+            <goals>
+              <goal>describe</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <format>git-Canyon-%s</format>
+        </configuration>
       </plugin>
+      <!-- Canyon End -->
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-- 
2.22.0

